app/src/main/res/values-nl/strings.xml

```diff
@@ -0,0 +1,1029 @@
+    <string name="developer_mode">Ontwikkelaarsmodus</string>
+    <string name="developer_mode_key">developer_mode</string>
+    <string name="activate_developer_mode">Ontwikkelaarsmodus inschakelen</string>
+    <string name="developer_mode_activation_message">U staat op het punt de ontwikkelaarsmodus in te schakelen. Deze modus geeft u toegang tot functies die nog niet officeel zijn vrijgegeven. Deze functies zijn mogelijk instabiel en kunnen bijwerkingen hebben of tot gegevensverlies leiden. Wees dus voorzichtig! Weet u zeker dat u deze modus wilt inschakelen?</string>
+    <string name="deactivate_developer_mode">Ontwikkelaarsmodus uitschakelen</string>
+    <string name="developer_mode_deactivation_message">U staat op het punt de ontwikkelaarsmodus uit te schakelen. De niet-vrijgegeven functies worden verborgen. Wilt u doorgaan?</string>
+    <string name="yes_take_chances">Ja, ik neem het risico</string>
+    <string name="no_stay_safe">Nee, toch maar niet</string>
+    <string name="yes">Ja</string>
+    <string name="no">Nee</string>
+    <string name="account">Account</string>
+    <string name="start_scanning_again">Opnieuw scannen</string>
+    <string name="play_sounds">Geluiden afspelen</string>
+    <string name="play_sounds_key">play_sounds</string>
+    <string name="play_sound_on_check_in_out">Geluid afspelen bij in- en uitchecken</string>
+    <string name="select_location">Kies een locatie</string>
+    <string name="mapbox_access_token">Mapbox-toegangssleutel</string>
+    <string name="access_token_required">U heeft een geldige toegangssleutel nodig</string>
+    <string name="family_name">Achternaam</string>
+    <string name="phone">Telefoonnummer</string>
+    <string name="social_media_profiles">Socialemediaprofielen</string>
+    <string name="profile">Profiel</string>
+    <string name="bio">Biografie</string>
+    <string name="about_us">Over ons</string>
+    <string name="about_us_description">Evenementbeheer-app voor organisatoren, gebruikmakend van het Eventyay-platform.\n\nDe kernfunctionaliteit van deze app bevat onder meer:\n Inloggen en registreren\n Evenementen organiseren\n Tickets aanmaken\n Bezoekers handmatig inchecken\n Bezoekers inchecken door hun QR-code te scannen\n Eenvoudige evenementstatistieken\n Eenvoudig ticketbeheer\n Evenementinformatie\n Bestelinformatie\n Organisatorinformatie</string>
+    <string name="github_repo">GitHub-repo</string>
+    <string name="developers">Ontwikkelaars</string>
+    <string name="github_repo_link">https://github.com/fossAzië/open-event-organizer-android</string>
+    <string name="github_developers_link">https://github.com/fossAzië/open-event-organizer-android/graphs/contributors</string>
+    <string name="third_party_licenses">Gebruikte externe bibliotheken</string>
+
+    <string-array name="timezones">
+        <item>Afrika/Abidjan</item>
+        <item>Afrika/Accra</item>
+        <item>Afrika/Addis_Abbeba</item>
+        <item>Afrika/Algerije</item>
```

Comment:
This should not be translated

Reply:
Why not?

Reply:
Because they're standard timezone names regardless of the language

Reply:
Hmm, with other projects, this is not an issue. But fair enough, I'll change it back then :)

Reply:
Done!

Reply:
Other projects like? AFAIK, you can't just send any translation of a timezone name to a date API, it won't understand it. I wished this was in code and not in strings or at least translatable="false" was set on it

Reply:
BRW, the build is failing after the addition of changes

Reply:
Why is it failing still?

Reply:
You can know by clicking on details

Reply:
It doesn't tell me anything useful. Seems like a few tests and stuff are failing, so that's something for the dev team to look at.

Reply:
Not really, build fails after these changes

Reply:
But why? I changed the timezone list back...